assets/js/search.js

```diff
@@ -79,7 +81,23 @@ export function findIn (elements, matcher) {
   }).filter(cleaner)
 }
 
-function search (nodes, value) {
+export function getParameterByName (name, url) {
+  if (!url) url = window.location.href
+  var param = name.replace(/[\[\]]/g, '\\$&')
+  var regex = new RegExp('[?&]' + param + '(=([^&#]*)|&|#|$)')
+  var results = regex.exec(url)
+  if (!results) return null
+  if (!results[2]) return ''
```

Comment:
@aphillipo Try to surround the `if` with braces even if is just one sentence. At least that's consistent with the rest of your code.